### DISCUSSION TITLE
Set font size of a label programmatically using NamedSize

### DISCUSSION THREAD

**Discussion by Dry-Tech** [CommunityToolkit/Maui#1155] [Q&A]
Hello,
how to set the font size of a label to NamedSize.Large in c#?

This way it does not work:
`new Label {
                    FontSize = NamedSize.Large
                };`

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
